scipy/sparse/_bsr.py

```diff
@@ -94,8 +94,10 @@ def __init__(self, arg1, shape=None, dtype=None, copy=False, blocksize=None):
                     if not isshape(blocksize):
                         raise ValueError(f'invalid blocksize={blocksize}')
                     if tuple(blocksize) != self.data.shape[1:]:
-                        raise ValueError('mismatching blocksize={} vs {}'.format(
-                            blocksize, self.data.shape[1:]))
+                        raise ValueError(
+                            f'mismatching blocksize={blocksize}'
+                            f'vs {self.data.shape[1:]}'
```

Comment:
You probably want a space either at the end of the first string or at the start of the second string. I typically put it at the start of the second string but that's personal preference. If the space isn't in either spot the `{blocksize}' output runs into the `vs`.